### ISSUE TITLE
{{notes}} does not invoke "Zotero Note Template" but is read from extensions.mdnotes.placeholder.notes

### ISSUE THREAD

**Issue by simonmaris** [argenos/zotero-mdnotes#140]
**Background:** I want to move my annotations before the metadata. I am using _Create full note export_.

I played around with the templates, and according to [the docs](https://argentinaos.com/zotero-mdnotes/docs/next/advanced/templates/single-file#zotero-note-template):

> If you include the {{notes}} placeholder in the Mdnotes Default Template, mdnotes will loop through the Zotero Notes of the selected items and add them one by one using this template.

This does not happen, instead (and this also makes sense), it uses _extensions.mdnotes.placeholder.notes_.

Therefore, by default it creates a List of all the items notes. 

These should be two different things. Maybe the _extensions.mdnotes.placeholder_ should instead be called **notesList**? 

**Desktop (please complete the following information):**
 - OS: [MacOS Big Sur 11.6]
 - Zotero version: [5.0.96.3]
 - Mdnotes version: [0.2.0-alpha4]
 - Zotfile version: [5.0.16]
 - BetterBibtex version: [5.5.4]

**Comment by simonmaris** [comment]
Hi and sorry for the delay.. I will try to be more specific..
This is the _Mdnotes Default Template.md_ that is used when I press _Create full note export_

```
---
aliases: ["{{yamltitle}}"]
---
# {{yamltitle}}
{{abstractNote}}

## Metadata

{{itemType}}
{{bookTitle}}
{{author}}
{{date}}
{{DOI}}
{{collections}}, {{tags}}, #zotero,  #literature-notes,  #reference
{{book}}
{{editor}}

### Links
{{pdfAttachments}}
{{localLibrary}}
{{cloudLibrary}}
```

It works fine. Notice there is no {{notes}} in here, the annotations (from the notes in zotero) just get appended at the end using the _Zotero Note Template_. 

Now my plan was to move the extracted annotations _before_ the ##Metadata section. Like i described above, the docs mac it sound like {{notes}} would do this, but that instead will just produce a markdown list of the available notes, not their content. 

I hope this clears things up. It's not super inconvenient and I will probably just keep the Metadata on top, I just wanted to point out the discrepancy between documentation and actual behaviour.. 

Thanks for your patience and this very useful plugin!

**Comment by argenos** [comment]
Looks like what you want is the Single-note exports: https://argentinaos.com/zotero-mdnotes/docs/next/advanced/templates/single-file

**Comment by simonmaris** [comment]
Mhm, I have indeed been using single-note export anyway and just double-checked both templates, they seem to be okay but it does not seem to work. I have added `{{notes}}` in the middle of my _MDnotes Default Template_. `{{notes}}` is basically doing what I would expect, it used the `extensions.mdnotes.placeholder.notes` setting, which in my case is the default:
```
{"content":"## Highlights and Annotations\n\n- {{field_contents}}", "field_contents": "{{content}}", "link_style": "wiki", "list_separator": "\n- "}
```
Why would `{{notes}}` do anything else if this placeholder setting is there? BTW: The actual annotation content is still there at the end of the generated file. (auto-appended)

I could paste all the extensions.mdnotes.* settings and the templates if you think that helps.
